src/App.js

```diff
@@ -460,6 +477,11 @@ function App() {
             <Typography variant="h3" gutterBottom align="center">
               Please Log In To Continue
             </Typography>
+            {auth0works || (
+              <Typography variant="h5" gutterBottom align="center">
+                Auth0 is not responding
+              </Typography>
+            )}
```

Comment:
If Auth0 is down, we should probably disable the login button along with this text?